charts/gateway-operator/values.yaml

```
@@ -2,6 +2,19 @@ image:
   repository: docker.io/kong/gateway-operator
   tag: 1.2
 
+extraContainers:
```

Let's rename it to `kubeRBACProxy`, so it will be clearer.

```suggestion
kubeRBACProxy:
```